src/tarballs/node.ts

```diff
@@ -1,12 +1,11 @@
 import {Errors, Interfaces} from '@oclif/core'
-import * as path from 'path'
+import * as path from 'node:path'
 import * as fs from 'fs-extra'
-import {pipeline as pipelineSync} from 'node:stream'
+import {pipeline} from 'node:stream/promises'
 import {log} from '../log'
 import {exec as execSync} from 'node:child_process'
 import {promisify} from 'node:util'
 import got from 'got'
-const pipeline = promisify(pipelineSync)
```

Comment:
Use the  promisified `pipeline` function provided by Node.js. This was introduced in Node.js 15.